tests/dataprep/test_dataprep_milvus_multimodal.sh

```diff
@@ -43,14 +44,16 @@ function build_docker_images() {
 function build_lvm_docker_images() {
     cd $WORKPATH
     echo $(pwd)
-    docker build --no-cache -t opea/lvm-llava:comps --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/third_parties/llava/src/Dockerfile .
+    dockerfile_name="comps/third_parties/llava/src/$1"
```

Comment:
You don't have a `comps/third_parties/llava/src/Dockerfile.openEuler` in the PR. Same issue for other tests. Suggest in the first pr, only integrate part of the test with Dockerfile.openEuler.